app/models/post.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
+
 class Post < ApplicationRecord
   belongs_to :user
```

Comment:
Set validations on this model and constraints in the database schema, I can submit an empty post because there are no validations nor constrains

Reply:
ok will do that